packages/ilmomasiina-frontend/src/modules/auditLog/actions.ts

```diff
@@ -34,37 +35,37 @@ export type AuditLogActions =
   | ReturnType<typeof auditLogLoadFailed>
   | ReturnType<typeof resetState>;
 
-export const getAuditLogs = (query: AuditLog.List.Query = {}) => async (
+export const getAuditLogs = (query: AuditLoqQuery = {} as AuditLoqQuery) => async (
   dispatch: DispatchAction,
   getState: GetState,
 ) => {
-  const { accessToken } = getState().auth;
-
   dispatch(auditLogQuery(query));
 
   const queryString = _.entries(query)
-    .filter(([, value]) => value)
-    .map(([key, value]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
+    .filter(([, value]) => !!value)
```

Comment:
No need to cast to boolean, `filter` does that for us
```suggestion
    .filter(([, value]) => value)
```

Reply:
Resolved in 8d88854